src/app/client/invite-employee/invite-employee.component.ts

```
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+
+@Component({
+  selector: 'app-invite-employee',
+  standalone: true,
+  templateUrl: './invite-employee.component.html',
+  styleUrls: ['./invite-employee.component.css'],
+  imports: [MatButtonModule],
+})
+export class InviteEmployeeComponent implements OnInit {
+  constructor() {}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)